src/sdk/model/vm/vm.ts

```diff
@@ -1274,6 +1274,27 @@ export class Vm extends Entity implements EntityWithPerfSamples {
         return new Task(apiTask);
       });
   }
+
+  /**
+   * Gets a new downloadable bundle (ZIP archive) of a collection of file snapshots using a Task uuid
+   *
+   * @param taskUuid {string} the uuid of the Task that originally created the file bundle
+   * @param downloadProgressCallback {function} optional progress callback if user wishes to receive progress events
```

Comment:
Nice. So an auth token isn't required for this one anymore?

Reply:
It does, the iland http class takes care of that for us with its ilandAxios interceptors